### ISSUE TITLE
YearlessDate objects cannot be serialized into migration file

### ISSUE THREAD

**Issue by nahuelange** [seddonym/django-yearlessdate#6]
makemigration with new fields raise an ValueError exception with this traceback:
```
Traceback (most recent call last):
  File "./manage.py", line 22, in <module>
    execute_from_command_line(sys.argv)
  File "/code/venv/lib/python3.6/site-packages/django/core/management/__init__.py", line 371, in execute_from_command_line
    utility.execute()
  File "/code/venv/lib/python3.6/site-packages/django/core/management/__init__.py", line 365, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/code/venv/lib/python3.6/site-packages/django/core/management/base.py", line 288, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/code/venv/lib/python3.6/site-packages/django/core/management/base.py", line 335, in execute
    output = self.handle(*args, **options)
  File "/code/venv/lib/python3.6/site-packages/django/core/management/commands/makemigrations.py", line 172, in handle
    self.write_migration_files(changes)
  File "/code/venv/lib/python3.6/site-packages/django/core/management/commands/makemigrations.py", line 210, in write_migration_files
    migration_string = writer.as_string()
  File "/code/venv/lib/python3.6/site-packages/django/db/migrations/writer.py", line 151, in as_string
    operation_string, operation_imports = OperationWriter(operation).serialize()
  File "/code/venv/lib/python3.6/site-packages/django/db/migrations/writer.py", line 110, in serialize
    _write(arg_name, arg_value)
  File "/code/venv/lib/python3.6/site-packages/django/db/migrations/writer.py", line 74, in _write
    arg_string, arg_imports = MigrationWriter.serialize(_arg_value)
  File "/code/venv/lib/python3.6/site-packages/django/db/migrations/writer.py", line 279, in serialize
    return serializer_factory(value).serialize()
  File "/code/venv/lib/python3.6/site-packages/django/db/migrations/serializer.py", line 203, in serialize
    return self.serialize_deconstructed(path, args, kwargs)
  File "/code/venv/lib/python3.6/site-packages/django/db/migrations/serializer.py", line 90, in serialize_deconstructed
    arg_string, arg_imports = serializer_factory(arg).serialize()
  File "/code/venv/lib/python3.6/site-packages/django/db/migrations/serializer.py", line 370, in serializer_factory
    "topics/migrations/#migration-serializing" % (value, get_docs_version())
ValueError: Cannot serialize: <djangoyearlessdate.helpers.YearlessDate object at 0x7fb228cc4eb8>
There are some values Django cannot serialize into migration files.
For more, see https://docs.djangoproject.com/en/2.0/topics/migrations/#migration-serializing

```

**Comment by seddonym** [comment]
Thanks for the bug report and sorry it's taken me a while to look at it.

I believe this might be happening because you're including a `YearlessDate` object in your field definition (perhaps as a default)? Could you confirm? It would be useful if you could include the field declaration you've used.

**Comment by johnfraney** [comment]
@seddonym I can reproduce this using defaults:

```python
from django.db import models
from djangoyearlessdate.helpers import YearlessDate
from djangoyearlessdate.models import YearlessDateField


class MyModel(models.Model):
    annual_start_date = YearlessDateField(default=YearlessDate(day=1, month=1))
    annual_end_date = YearlessDateField(default=YearlessDate(day=31, month=12))
```

**Comment by seddonym** [comment]
I thought as much! Great, I'll look into it.

**Comment by seddonym** [comment]
This should now be fixed if you upgrade to version 1.3.
